lighthouse-core/lib/i18n/i18n.js

```diff
@@ -56,9 +56,13 @@ const UIStrings = {
   displayValueMsSavings: 'Potential savings of {wastedMs, number, milliseconds}\xa0ms',
   /** Label for a column in a data table; entries will be the URL of a web resource */
   columnURL: 'URL',
-  /** Label for a column in a data table; entries will be the size of a web resource in kilobytes. */
+  /** Label for a column in a data table; entries will be the size of something resource (generic). */
   columnSize: 'Size',
+  /** Label for a column in a data table; entries will be the transfer size (on the wire) of a web resource in kilobytes. */
+  columnResourceSize: 'Resource Size',
```

Comment:
in https://github.com/GoogleChrome/lighthouse/issues/10357#issuecomment-591075172 it looks like the consensus might have been "File Size"? Even though we use resourceSize internally, I can see the argument for clearly communicating with end users

Reply:
In web parlance, is "file" really more clear than "resource"? Are files a thing, really?

Reply:
we decided resource size. also, btw CDT says resource size in network panel, so thats a strong reason to go with resource.